nodejs/docs/api/class-fixtures.mdx

```diff
@@ -74,7 +74,7 @@ Name of the browser that runs tests. Defaults to `'chromium'`. Useful to [annota
 }>
 <TabItem value="ts">
 
-```ts
+```js
 test('skip this test in Firefox', async ({ page, browserName }) => {
```

Comment:
This delta looks off.

Reply:
It is deliberate - it now respects what's set in the source.